CHANGELOG.md

```diff
@@ -1,5 +1,9 @@
 # Change Log
 
+## [release-26] - 2018-12-17
```

Comment:
the markdown link for this is missing from the bottom of the file.

Reply:
doh!